roles/kernelci-storage/templates/kci-storage.conf

```diff
@@ -0,0 +1,78 @@
+server {
+    listen *;
+    listen [::];
+
+    server_name {{ kci_storage_fqdn }};
+    root /var/www/images/kernel-ci;
+    charset utf-8;
+
+    access_log /var/log/nginx/{{ kci_storage_fqdn }}-access.log combined buffer=16k;
+    error_log /var/log/nginx/{{ kci_storage_fqdn }}-error.log crit;
+
+    if ($host != "{{ kci_storage_fqdn }}") {
+        return 403;
+    }
+
+    # For letsencrypt auto-renew to work.
+    location /.well-known/ {
+        root /usr/share/nginx/html/{{ kci_storage_fqdn }}/;
+    }
+
+    location / {
+        if (-f $document_root/maintenance.html) {
+            return 503;
+        }
+
+{% if ansible_distribution != "CentOS" %}
+        fancyindex on;
+        fancyindex_exact_size off;
+{% endif %}
+    }
+}
+
+{% if role == "production" %}
+server {
+    listen *:443 http2 ssl;
+    listen [::]:443 http2 ssl;
+
+    ssl_certificate /etc/letsencrypt/live/{{ kci_storage_fqdn }}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/{{ kci_storage_fqdn }}/privkey.pem;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:HIGH:!ADH:!AECDH:!aNULL:!MD5:!eNULL";
+    ssl_dhparam /etc/ssl/certs/dhparam.pem;
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_session_timeout 10m;
+    ssl_session_cache shared:SSL:50m;
+    ssl_stapling on;
+    resolver 8.8.8.8 8.8.4.4 [2001:4860:4860::8888] [2001:4860:4860::8844] 213.133.98.98 213.133.99.99 [2a01:4f8:0:a0a1::add:1010] [2a01:4f8:0:a102::add:9999] ipv6=on;
```

Comment:
There is a variable that defines the list of resolver (usually left as a secret), since you might want not to show them to the outside.

Look at the template for the api.kernelci.org configuration, there is something like this:
```
{% if "ssl_stapling_resolver" is defined %}
    ssl_stapling on;
    resolver {{ ssl_stapling_resolver }} ipv6=on;
    resolver_timeout 3s;
{% endif %}
```

Reply:
ok